test/test_testing.py

```diff
@@ -1308,7 +1308,7 @@ def test_matching(self):
 
     def test_mismatching_ccol_indices_msg(self):
         actual_ccol_indices = (0, 1, 2)
-        actual_row_indices = (1, 0)
```

Comment:
Otherwise it fails with unsorted indices, right?

Reply:
Yes, correct. This changeset fixes incorrect test data.